lib/graphql/enum_type.rb

```diff
@@ -67,6 +67,7 @@ def validate_non_null_input(value_name)
     # @param value_name [String] the string representation of this enum value
     # @return [Object] the underlying value for this enum value
     def coerce_non_null_input(value_name)
+      return nil unless @values_by_name.keys.include?(value_name)
       @values_by_name.fetch(value_name).value
```

Comment:
I did that with a `try!(:value)` and then realized that `try!` is rails-specific; I suppose I could guard it with a ternary.